source/Stargate-Model/NeoJSONWriter.extension.st

```diff
@@ -0,0 +1,35 @@
+Extension { #name : #NeoJSONWriter }
+
+{ #category : #'*Stargate-Model' }
+NeoJSONWriter >> mapZincEvent [
```

Comment:
I think this needs better tests, we need to be sure that every kind of object the event includes is properly mapped (like the abstractions we have in Hyperspace: WebLink, EntityTags, etc).